### ISSUE TITLE
Somehow advertise when state upload fails due to size or maximum allowed size

### ISSUE THREAD

**Issue by bdpedigo** [CAVEconnectome/NeuroglancerJsonServer#3]
I often get 

`HTTPError: 502 Server Error: Bad Gateway for url:...`

when trying to upload a JSON that is too big. This error is unintuitive, would be nice to get something that states this explicitly.

Even better, would be nice to know how big my JSON can be, but @ceesem says maybe that is complicated because of compression/encoding?

**Comment by fcollman** [comment]
Yes I think it is complicated because of compression.. really i would like to see us finish moving the storage of the json out of datastore and into cloud storage so that the size can be larger
